system-tests/projects/react17/src/UsingLegacyMount.cy.jsx

```diff
@@ -1,22 +1,24 @@
-import React from 'react'
-import { mount } from 'cypress/react'
+// import React from 'react'
```

Comment:
I do not have a good solution here. This test cannot execute - in fact, it doesn't even compile,. since `cypress/react` assumes `react-dom/client` exists, which is a React 18 only API. This causes Vite to throw a compilation error.

Reply:
Interesting idea... I haven't `proxyquire` - It might be possible, we'd need to make sure it works for both bundlers... at this point, I feel like a clean migration UI might even be more simple and cleaner, without adding too much technical complexity. As we also discussed earlier, we won't rush this for Cy 12, so maybe we can spend a bit more time thinking about it.